public/app/plugins/panel/table/specs/renderer.test.ts

```diff
@@ -451,6 +477,45 @@ describe('when rendering table with different patterns', () => {
   );
 });
 
+describe('when rendering cells with different alignment options', () => {
+  it.each`
+    align        | expected
+    ${'default'} | ${'<td style="text-align:default">42</td>'}
+    ${'left'}    | ${'<td style="text-align:left">42</td>'}
+    ${'center'}  | ${'<td style="text-align:center">42</td>'}
+    ${'right'}   | ${'<td style="text-align:right">42</td>'}
+    ${'justify'} | ${'<td style="text-align:justify">42</td>'}
+  `('$align-aligned columns should be formatted with correct style', ({ align, expected }) => {
+    const table = new TableModel();
+    table.columns = [{ text: 'Time' }, { text: align }];
+    table.rows = [[0, 42]];
+
+    const panel = {
+      pageSize: 10,
+      styles: [
+        {
+          pattern: 'Time',
+          type: 'date',
+          format: 'LLL',
+          alias: 'Timestamp',
+        },
+        {
+          pattern: `/${align}/`,
+          align: align,
+          type: 'number',
+          unit: 'none',
+        },
+      ],
+    };
+
+    //@ts-ignore
+    const renderer = new TableRenderer(panel, table, 'utc', sanitize, templateSrv);
+    const html = renderer.renderCell(1, 0, 42);
+
+    expect(html).toBe(expected);
+  });
+});
+
```

Comment:
Looks great! Would also be great to have tests for the combination of styles as the code has been changed to an array of styles.